### PR TITLE
Bump upstream versions

### PR DESCRIPTION
**Pull request by Tobianas**
Adopt:
- odlparent-11.0.2
- infrautils-4.0.2
- yangtools-9.0.2
- mdsal-10.0.3
- controller-6.0.3
- aaa-0.16.4
- netconf-4.0.3
- bgpcep-0.18.3

LIGHTY-113

### CODE REVIEW

**Review by ihrasko — CHANGES_REQUESTED** [review]

File: lighty-core/dependency-versions/pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
             <dependency>
                 <groupId>org.opendaylight.odlparent</groupId>
                 <artifactId>odlparent</artifactId>
-                <version>11.0.1</version>
```
[the remark this answers is not in the document]
Comment (Tobianas) [reply lighty-core/dependency-versions/pom.xml:28]: Changed in all other .pom files

**Review by ihrasko — CHANGES_REQUESTED** [review]
odlparent 11.0.2
infrautils 4.0.2
yangtools 9.0.2
mdsal 10.0.3
controller 6.0.3
aaa 0.16.4
netconf 4.0.3
bgpcep 0.18.2

**Comment by ihrasko** [comment]
Please bump bgpcep to 0.18.3.

**Review by ihrasko — APPROVED** [review]

**Comment by ihrasko** [comment]
OK, done.

**Pull request merged**